src/apps/loan-list/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import get from "lodash.get";
+import { LoanV2 } from "../../core/fbs/model/loanV2";
+import dayjs from "dayjs";
+
+export const removeLoansWithDuplicateDueDate = (
+  date: string,
+  list: LoanV2[],
+  filterByPath: string
+) => {
+  return list.filter(
+    (material: LoanV2) => get(material, filterByPath) === date
+  );
+};
+
+export const getAmountOfRenewableLoans = (list: LoanV2[]) => {
+  return list.filter(({ isRenewable }) => isRenewable).length;
+};
+
+export const formatDate = (date: string) => {
+  return dayjs(date).format("DD-MM-YYYY");
+};
+
+export const materialOverdue = (date: string) => {
```

Comment:
Really a minor, but I think if a function returns a boolean it should be called something like:
`materialIsOverdue` to emphasize the purpose of it